apps/whois/requirements

```diff
@@ -0,0 +1,2 @@
+perl-DateTime.x86_64 : Date and time object
+whois-5.1.1-2.el7.x86_64  : /usr/bin/whois binary
```

Comment:
Please add it to the PLUGIN DESCRIPTION if you feel like it but don't create a dedicated file for it.

Reply:
done.